integration/release/release.go

```diff
@@ -0,0 +1,61 @@
+// +build k8srequired
+
+package release
```

Comment:
Unsure about the package name, I couldn't think of anything better. Suggestions welcome.

Reply:
I'm fine with the name, but maybe change the functions? I think `release.WaitForStatus` would be better than `release.WaitForReleaseStatus`. Same for version.

Reply:
Good shout, on it.

Reply:
Done.